### PR TITLE
built-ins docs

### PR DESCRIPTION
**Pull request by gluax**
Documentation about built-in imports.

### CODE REVIEW

**Comment by gluax** [comment]
Deadlink error in CI will be resolved when stdlib is merged shortly!

File: documentation/developer/language/10_imports.md [hunk 1]
```diff
@@ -45,10 +47,44 @@ function main() {
...
+
+    return result;
+}
+```
+
+You can see more about these built-in types [here](https://github.com/AleoHQ/leo/tree/master/stdlib).
+
+### Automatic Built-ins
+
+Some built-ins are automatically imported to the language.
```
Comment (acoglio) [inline documentation/developer/language/10_imports.md:74]: imported into every program.

File: documentation/developer/language/10_imports.md [hunk 2]
```diff
@@ -45,10 +47,44 @@ function main() {
...
+    let s: string = "Hello, string type alias!";
+}
+```
+
 ## Package Paths
+
 Nested import paths can be accessed using dot `.` syntax.
 
-**Example:**
+** Example:**
```
Comment (acoglio) [inline documentation/developer/language/10_imports.md:87]: Stray space.

File: documentation/developer/language/10_imports.md [hunk 3]
```diff
@@ -45,10 +47,44 @@ function main() {
...
+    return result;
+}
+```
+
+You can see more about these built-in types [here](https://github.com/AleoHQ/leo/tree/master/stdlib).
+
+### Automatic Built-ins
+
+Some built-ins are automatically imported to the language.
+For Example, the type alias `string` is imported and usable with no import statement necessary.
```
Comment (acoglio) [inline documentation/developer/language/10_imports.md:75]: example

**Review by acoglio — APPROVED** [review]
Looks good. I added a few suggestions inline. Also, I think this PR should be to the `next-release` branch.

**Comment by gluax** [comment]
@acoglio you're right. Forgot that we made that we are starting that already!

**Comment by collinc97** [comment]
deprecated

**Pull request closed without merging**